.github/workflows/release.yaml

```
@@ -58,7 +58,7 @@ jobs:
       - name: Upload MacOS build
         uses: actions/upload-artifact@v3
         with:
-          name: dependencies.sources-arm64
+          name: DependenciesSourceCode.zip
           path: ./downloads/dependency-sources*
```

Maybe we can make it more specific so that it can be easier to follow?

```suggestion
          path: ./downloads/dependency-sources.tar.gz
```

---

Updated.